_includes/main-title.html

```diff
@@ -1,5 +1,22 @@
 <div class="jumbotron background-white header-tile">
-  {% include nav.html %}
+  <!-- This page has special styling with the CTA that is why we're not using the nav.html -->
+  <div class="row m-0 justify-content-between">
+    <div>
+      <button class="navbar-toggler collapsed hamburger-background light-purple-background align-left" type="button"
+        data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault"
+        aria-expanded="false" aria-label="Toggle navigation" onclick="on()">
+        <span class="icon-bar top-bar color-black"></span>
+        <span class="icon-bar middle-bar color-black"></span>
+        <span class="icon-bar bottom-bar color-black"></span>
+      </button>
+    </div>
+    <div>
+      <a href="/">
+          <img src="{{ 'assets/img/UX/cscbus_logo_square.svg' | relative_url }}" align="right" style="height: 75px;" alt="CSC Home" />
+        </a>
+    </div>
+  </div>
+
```

Comment:
Maybe we can refactor to remove this but the ordering really messes with things if we're not expecting that standardized CTA.

Reply:
I think it's ok to have it in two places if we have to. So long as all the other spots use the refactored nav, that'll save us some coding.

unless we could do something clever like put a placeholder between them like an empty div?
```
<div class="mb-2">&nbsp;</div>
```